ui/src/variables/reducers/index.ts

```diff
@@ -78,6 +78,13 @@ export const variablesReducer = (
         return
       }
 
+      case 'SELECT_VALUE': {
+        const {contextID, id, value} = action.payload
+        draftState.values[contextID].values[id].selectedValue = value
```

Comment:
We might want to check that this path exists before trying to set it (e.g. that `values[contextID]` exists and `values[id]` exists and so on).

Reply:
would something like `if (!!draftState.values[contextID].values[id].selectedValue)` work?

Reply:
I think that will still throw an error if the `values[contextID]` part doesn't exist, or if the `values[id]` part doesn't exist (would see something like `Error: can't get selectedValue of null`)

How about:

```
        const {contextID, id, selectedValue} = action.payload

        const valuesExist = !!get(
          draftState,
          `values.${contextID}.values.${id}`
        )

        if (!valuesExist) {
          return
        }

        draftState.values[contextID].values[id].selectedValue = selectedValue
```
since what